cypress/helpers/period.js

```diff
@@ -7,11 +7,49 @@ const openPeriod = (label) => {
     }
 }
 
-const selectFixedPeriod = ({ label, period }) => {
+const selectFixedPeriod = ({ label, period, selected }) => {
     openPeriod(label)
     cy.contains('Choose from presets').click()
     cy.contains('Fixed periods').click()
-    if (period.type) {
+
+    if (selected) {
+        // confirm that selected item is in the correct list in the transfer
+        if (
+            period.type &&
+            !cy
```

Comment:
I'm not sure if you can actually do this (use the value returned from `cy` in an `if` statement) or if this just works by chance.
Regardless, I'd say given that `selected` is only used in one place, instead of making `selectFixedPeriod` overly complicated, just do everything manually directly in `createTrackedEntity.cy.js` instead. 
Preferably we keep the test files as simple as possible to avoid misuse and issues with the tests themselves (i.e. unlike "regular code", not everything has to go into a helper function for test code)

Reply:
Good catch. Code changed accordingly.